src/main/java/http/controllers/LogsController.java

```diff
@@ -0,0 +1,25 @@
+package http.controllers;
+
+import http.IO.file.InvalidPathException;
+import http.request.HttpRequest;
+import http.response.HttpResponse;
+import http.status.HttpStatus;
+import http.status.InvalidStatusCodeException;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+
+public class LogsController implements IController {
+
+    @Override
+    public HttpResponse generateResponse(HttpRequest httpRequest) throws IOException, InvalidPathException, InvalidStatusCodeException {
+            return handleAuthorized(httpRequest);
```

Comment:
how come this is extracted into another function? couldnt this just be inlined?

Reply:
That makes sense. I'll inline it.